src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,41 @@
+package calculator;
+
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    private String userInput;
+    private int result = 0;
+    private Operator currentOperator = Operator.PLUS;
```

Comment:
초기화를 한 특별한 이유가 있을까요? 
```suggestion
    private Operator currentOperator;
```
불필요한 구문 인 것 같습니다.